roles/arrstack/templates/buildarr/instances/sonarr-instances.yml.j2.example

```diff
@@ -357,11 +358,12 @@ sonarr:
 
     # Sonarr instance for grabbing anime shows.
     sonarr-anime:
-      api_key: "{{ arrstack_instance_apikeys['sonarr-anime'] }}"
+      hostname: {{ hostname_sonarr | default('localhost') }}
+      api_key: "{{ arrstack_instance_apikeys['sonarr-anime'] if arrstack_instance_apikeys is defined and 'sonarr-anime' in arrstack_instance_apikeys else 'ansible_api_key_not_found' }}"
```

Comment:
Putting in an incorrect value when something is not found is an anti-pattern I believe.

Instead, let Ansible raise an error.

```suggestion
      api_key: "{{ arrstack_instance_apikeys['sonarr-anime'] }}"
```